raiden/network/rpc/client.py

```diff
@@ -830,7 +830,7 @@ def poll(self, transaction_hash: bytes):
                 if block_number >= confirmation_block:
                     return transaction
 
-            gevent.sleep(1.0)
+            gevent.sleep(10)
```

Comment:
This can slow down a bit regtest but we can modify it later

Reply:
We could add that to settings or a function that calculates that from the current network or something like that.